bin/wmcoreD

```diff
@@ -235,6 +238,16 @@ def startup(configFile):
         else:
             print('Path for daemon file does not exist!')
             sys.exit(1)
+        # write into component area process status information
+        cpath = os.path.join(compDir, "threads.json")
+        if os.path.exists(cpath):
+            os.remove(cpath)
+        cxml = os.path.join(compDir, "Daemon.xml")
```

Comment:
This line is dup of L231 (other than the variable name).

Reply:
ok, done